source/drivers/csharp.txt

```diff
@@ -127,6 +134,12 @@ MongoDB Compatibility
      - .NET 4.5
      - .NET Core
 
+   * - Version 2.4
```

Comment:
This isn't appearing on staging, currently -- not sure why. Assuming it does build correctly, LGTM!